### ISSUE TITLE
Failed to setup : cannot read property `width`of undefined

### ISSUE THREAD

**Issue by memelie** [highcharts/highcharts-react-native#5]
Hi there!

I am trying to setup the package as indicated in the readme, but this error stills happen (running iOS, did not try Android yet): 

> Unhandled JS Exception: TypeError: Cannot read property 'width' of undefined
> This error is located at:
>     in HighchartsReactNative 

In my package.json:
```
@highcharts/highcharts-react-native": "^1.0.0",
    "react": "^16.8.3",
    "react-native": "^0.59.8",
```

Code snippet:
```
import HighchartsReactNative from '@highcharts/highcharts-react-native/src/HighchartsReactNative';
...
render() {  
...
return (
...
<HighchartsReactNative
          options={{
            series: [{
              data: [1, 3, 2],
            }],
          }}
          style={{
            flex: 1,
            width,
            height: 500,
          }}
        />
...
);
}
```

Note that `import HighchartsReactNative from '@highcharts/HighchartsReactNative';` results in an error as the package is not found.

Any help would be appreciated :)

**Comment by jackcb123** [comment]
@memelie I saw this too. You need to make sure you are using the react-native Stylesheet for your styles that you are passing in

**Comment by sebastianbochan** [comment]
Hi @memelie,
One of the reasons can be missing Stylesheet like @jackcb123  said. Please confirm if it solved your issue.

**Comment by memelie** [comment]
@jackcb123 @sebastianbochan thanks for your answers!
It solved the problem. I also noticed I have written `style=` instead of `styles=`.

Unfortunately, now I am facing a blank screen.

**Comment by sebastianbochan** [comment]
Do you receive any errors or something? Do you setup expo or different environment?

**Comment by ssuchanowski** [comment]
I think to avoid confusion `styles` should be renamed to `style`

**Comment by memelie** [comment]
@sebastianbochan thanks for your reply. No errors nor warning, just a white area. I am directly building from my project workspace in xCode (v10.2.1), I use cocoapods for some packages. No expo setup.

**Comment by sebastianbochan** [comment]
Hi @memelie,
Have you tried used @r-bman solution from the ticket https://github.com/highcharts/highcharts-react-native/issues/2 ?

**Comment by r-bman** [comment]
Agree with @ssuchanowski, I also had this issue and have renamed `styles` to `style` in my local version.

**Comment by memelie** [comment]
@sebastianbochan solution from the ticket #2  worked. Thanks all for your help. Agreed it's not a perfect solution.

**Comment by sebastianbochan** [comment]
Im just preparing new version, hope that will be released this week (including HC 7.1.2).

Thank you both for helping.
